### DISCUSSION TITLE
Using NextAuth in a browser extension

### DISCUSSION THREAD

**Discussion by mihainsto** [nextauthjs/next-auth#2864] [Help]
Hi there, is it possible? thanks

**Comment by balazsorban44** [comment]
Would need much more info to be able to answer. What's the use case, how do you think `next-auth` could help, etc.

**Reply by mihainsto** [reply]
I am having an api that is already using NextAuth, and I want to build a browser extension that uses the same api user accounts etc.   The browser extension I am building it also with next, but I am exporting the pages as static ones with ‘next export’ and the regular NextAuth way does not work.

**Reply by balazsorban44** [reply]
NextAuth.js relies on a secure backend being available at runtime, next export won't work in that case, I'm sorry.

**Reply by balazsorban44** [reply]
I don't know if #2294 would change that in the future.

**Reply by mihainsto** [reply]
Any other way to make NextAuth work on static pages?

**Reply by balazsorban44** [reply]
no, as said, it requires a runtime for checking/setting cookies, etc.
